docs/config.c

```diff
@@ -1615,7 +1616,7 @@
 ** entry when expanding the alias.  The default value
 ** will return the string up to the first "," encountered.
 ** If the GECOS field contains a string like "lastname, firstname" then you
-** should set it to "\fC.*\fP".
+** should set it to "\fI.*\fP".
```

Comment:
I would consider this a literal, so bold, I think.  Also, since the config should include the quotes, I'd also make them bold.